### ISSUE TITLE
TypeError: Cannot use 'in' operator to search for 'counts' in undefined

### ISSUE THREAD

**Issue by carlosduclos** [apache/couchdb-nano#22]
[Original issue](https://github.com/dscape/nano/issues/313)

__kanongil__
Regression in 6.2.0 from c03b552. The patch breaks lookups that has neither qs or callback set.

__jo__
Thanks for catching this!

__WaldoJeffers__
Yup, I have the same issue. For example, if do db.viewWithList('design','view','list').pipe(...), I get this error : TypeError: Cannot use &#39;in&#39; operator to search for 'counts' in undefined.

__MatthieuLemoine__
I also have this issue when using viewWithList as @WaldoJeffers.

__bednie__
Still seeing this. Would a PR help?

**Comment by bedney** [comment]
This bug has been fixed in PR: https://github.com/apache/couchdb-nano/pull/5

Feel free to close.

**Comment by devtanc** [comment]
@bedney Yes, this may have been fixed, but it's still happening. From what I can tell, it's most likely because of confusion on the part of the users as to what needs to be included in the `params` object. I got the issue when I tried to simply pass in a string instead of an object, but the documentation is unclear on what is expected.

**Comment by wprater** [comment]
this happens with promises as well.
